checkstyle/src/main/resources/nms_checks.xml

```diff
@@ -25,5 +25,24 @@
             <property name="format" value="^UTF-8$"/>
             <message key="forbid.UTF8.string.literal" value="Use ''java.nio.charset.StandardCharsets.UTF_8.name()'' instead of ''{0}''" />
         </module>
+
+        <!--
+            Ensure the imports are in a defined order, otherwise different IDEs enforce different results, which
+            causes unnecessary git diffs
+        -->
+        <module name="ImportOrder">
+            <property name="groups" value="java,javax,org,com,*" />
```

Comment:
do you want a separate group for org.opennms?
i assume idea can do this.

Reply:
When I originally created the import rules for Idea I refactored them from what was already there in OpenNMS + the default rules from eclipse. 

I don't really have a strong opinion about this, except * imports on non static packages.
I just like to have not to all the time worry about the imports when committing or reviewing code, as it is kinda spam (-:
So any (sane) order is fine for me (-: